charts/lomas/values.yaml

```diff
@@ -66,8 +69,11 @@ lomas-server:
             contrib: True
             floating_point: True
             honest_but_curious: True
-
-
+        private_db_credentials:
```

Comment:
I would keep this to empty list by default, since there are no private db that use these credentials.